app/views/surveys/_form.html.haml

```diff
@@ -7,5 +7,14 @@
     = f.label :group
     = f.text_field :group, placeholder: "Survey For"
 
+  %div.survey-form-title
```

Comment:
`%div.survey-form-title` can be written as `.survey-form-title` since `%div` is implicit